services/data/input/query_data.json

```diff
@@ -3,10 +3,10 @@
         "columns_required": ["name", "address", "curriculum__abbreviation"]
     },
     "college": {
-        "columns_required": ["name", "address"]
+        "columns_required": ["name"]
```

Comment:
The country has been added to the college model instead of the address. Please update it here as well.

Reply:
The change has already been made in previous commits. Only the "name" column has been kept.